libs/core/src/operators/backoff.pipe.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { Action } from '@ngrx/store';
+import { Observable, of } from 'rxjs';
+import { hot } from 'jasmine-marbles';
+import { switchMap, map } from 'rxjs/operators';
+
+import { backoff } from './backoff.pipe';
+declare var somethingThrowsError;
+
+describe('Core | Operators | Backoff', () => {
+  let actions$: Observable<Action>;
+	const action1: Action = {type: 'type1'};
+
+	it(`should retry the observable when the observable throws an error
+	    should pass through when the observable does not throw an error`, () => {
```

Comment:
Is this test name a mistake?

Reply:
It might be, but I'm not aware that it is